pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -118,6 +120,30 @@ func (n *nsLifecycle) ensurePRTBAddToNamespace(ns *v1.Namespace) (bool, error) {
 			return false, errors.Wrapf(err, "couldn't ensure binding %v in %v", prtb.Name, ns.Name)
 		}
 	}
+
+	prtbSet := map[string]string{}
+	prtbList := n.m.prtbIndexer.List()
```

Comment:
THis is going to get ALL prtbs across ALL clusters. could be a big list. shouldnt do this